template/template.go

```diff
@@ -16,19 +16,22 @@ package template
 import (
 	"bytes"
 
-	html_tmpl "html/template"
-	text_tmpl "text/template"
```

Comment:
I couldn't find a single example in the stdlib, outside of weird, close-to-C code.
Might be, of course, that you don't really need named imports in the stdlib.

Of course, there is the exception of something_test, but that's special and ever more of a reason to avoid underscores in normal package (re-)names.

So I'd like to avoid underscores until proven otherwise.